flaml/autogen/agentchat/groupchat.py

```diff
@@ -0,0 +1,204 @@
+import sys
+from typing import Dict, List, Optional, Union
+import random
+from .agent import Agent
+from .responsive_agent import ResponsiveAgent
+
+
+class Room(ResponsiveAgent):
```

Comment:
it's unclear on the expected behavior over the following example if `Room` is an `ResponsiveAgent`
- `Room.Send` and `Room.Receive` from an external agent
- `Room.Send` and `Room.Receive` from other room
- nested `Room` agent inside `Room`

Reply:
Very good point. I suggested the name "Room" but neglected the issues with "Room" inside the "Room". Let's discuss offline.